src/hooks/useToasts.ts

```diff
@@ -24,16 +31,21 @@ export const ToastsContext = createContext<UseToasts>({} as UseToasts);
 export default function useToasts(): UseToasts {
   const [toasts, setToasts] = useState<Toast[]>([]);
 
-  const addToast = useCallback((title: string, body: ToastBody) => {
-    setToasts((oldToasts) => [
-      ...oldToasts,
-      {
-        id: window.crypto.randomUUID(),
-        title,
-        body,
-      },
-    ]);
-  }, []);
+  const addToast = useCallback(
+    (title: string, body: ToastBody, autohide?: boolean, delay?: number) => {
```

Comment:
Prefer avoiding boolean args, as seeing true or false in a function call doesn't tell you much without looking at the function call definition.

How about:
```ts
addToast(
  title: string,
  body: ToastBody,
  { autohide = false }: { autohide?: boolean } = {}
)
```